### ISSUE TITLE
R CMD check fails with R-devel_2019-03-11

### ISSUE THREAD

**Issue by ressy** [ShawHahnLab/chiimp#27]
Running `R CMD check chiimp_0.2.2.tar.gz` fails on recent development versions of R:

```
...
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking whether package ‘chiimp’ can be installed ... ERROR
Installation failed.
See ‘/data/home/jesse/dev/chiimp-r-ver/chiimp.Rcheck/00install.out’ for details.
* DONE

Status: 1 ERROR
See
  ‘/data/home/jesse/dev/chiimp-r-ver/chiimp.Rcheck/00check.log’
for details.
```

The file chiimp.Rcheck/00install.out shows:

```
* installing *source* package ‘chiimp’ ...
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
ERROR: hard-coded installation path: please report to the package maintainer and use ‘--no-staged-install’
* removing ‘/data/home/jesse/dev/chiimp-r-ver/chiimp.Rcheck/chiimp’
```

This is causing CI via Travis to fail for the "devel" and "bioc-devel" jobs.  it's fine on the latest stable release, though.  Is this a bug in the development version or a more stringent check that didn't previously exist?

**Comment by ressy** [comment]
It's a more stringent check.  It now looks for evidence of absolute paths hardcoded into the byte-compiled objects, and that actually does exist in the form of `test_data$f.locus_attrs`.  That isn't even necessary to keep once the data is loaded, so a simple fix may be to just remove the path from the list entirely.

**Comment by ressy** [comment]
That looks like it'll work fine, but there are other problems with that same object, notably that `sample.int` no longer produces the same output given the same `set.seed(0)` call beforehand, and affects the RNG state for anything that follows.  A simple fix for now may be to use RNGversion to control the behavior, like `RNGversion(3.5.3)`.

**Comment by ressy** [comment]
Fixed in 515f8edad87d26aa851977572cd432b67eca3c69.
